### DISCUSSION TITLE
Unable to connect topic queue with below configuration

### DISCUSSION THREAD

**Discussion by pushparaj-raster** [arkmq-org/arkmq-org-broker-operator#913] [General] [answered]
Hi,
I’m trying to connect an ActiveMQ Artemis topic queue with the following configurations. Based on my settings, I don’t want to create dynamic queues when clients establish new connections. However, after applying this configuration, I’m unable to connect to my topic queue. Could you please explain why this is happening and guide me on resolving this issue?

![image](https://github.com/artemiscloud/activemq-artemis-operator/assets/77319029/25386a7c-fdcc-4bf5-9557-92e0caf87aec)
![image](https://github.com/artemiscloud/activemq-artemis-operator/assets/77319029/4d4c9a65-3cae-47af-bc3b-1b63f1713b5b)
![image](https://github.com/artemiscloud/activemq-artemis-operator/assets/77319029/8d271f37-fb4d-4715-8b38-a6a6fc9d23c7)

Here is my sample java code to connect this topic queue 
`String broker = "tcp://172.17.1.104:61616"; 
        String clientId = "demo_client";
        String topic = "ONEEMR/PUSH_NOTIFICATION_TOPIC"; 

        try {
            MqttClient client = new MqttClient(broker, clientId, new MemoryPersistence());
            MqttConnectOptions options = new MqttConnectOptions();
            client.connect(options);
            client.subscribe(topic, (currentTopic, msg) -> {
                String receivedMessage = new String(msg.getPayload());
                System.out.println("Received message: " + receivedMessage);
            });

            while (true) {
                Thread.sleep(1000);
            }
        } catch (MqttException e) {
            e.printStackTrace();
            System.err.println("MQTT Exception: " + e.getMessage());
        } catch (InterruptedException e) {
            Thread.currentThread().interrupt();
            System.err.println("Thread interrupted: " + e.getMessage());
        }`
        
        
**Thanks in advance**...😊

**Comment by brusdev** [comment, marked as answer]
@pushparaj-raster this is a question for Apache ActiveMQ Artemis, you should send an email to [users@activemq.apache.org](mailto:users@activemq.apache.org).

**Comment by pushparaj-raster** [comment]
Okay

On Mon, 29 Apr 2024 at 19:37, Bruscino Domenico Francesco <
***@***.***> wrote:

> @pushparaj-raster <https://github.com/pushparaj-raster> this is a
> question for Apache ActiveMQ Artemis, you should send an email to
> ***@***.***
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/artemiscloud/activemq-artemis-operator/discussions/913#discussioncomment-9263344>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASN4W5PG7KOV4JQHZ56OFBLY7ZH3VAVCNFSM6AAAAABG6ABIFSVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TENRTGM2DI>
> .
> You are receiving this because you were mentioned.Message ID:
> <artemiscloud/activemq-artemis-operator/repo-discussions/913/comments/9263344
> @github.com>
>


-- 
*------------------------*
*Thanks & Regards,*
*Pushparaj Chinnathambi*
Team Lead
*Raster Images Pvt. Ltd.*
Salem 636 004, TN, India
E: ***@***.***

**Comment by brusdev** [comment]
Question sent to users@activemq.apache.org, for further details see https://lists.apache.org/thread/5kq1qoqt6dl49c21j0xm8h7z7tmqm66s
